bcache/get.go

```diff
@@ -251,7 +264,7 @@ func (p *parser) getPriorityStats() PriorityStats {
 }
 
 // GetStats collects from sysfs files data tied to one bcache ID.
-func GetStats(uuidPath string) (*Stats, error) {
+func GetStats(uuidPath string, priorityStats bool) (*Stats, error) {
```

Comment:
This one is also an exported function, so it's possible that this change would break some users.  However, it looks like node_exporter doesn't use this directly, so I think this change should be ok.

Reply:
We can do wrappers here too, but for me it looks like adding complexity to the internal api. Also, not sure it will be used outside of this library.
Maybe, it is better to change it to getStats.